src/System.Diagnostics.Process/pkg/System.Diagnostics.Process.pkgproj

```diff
@@ -3,20 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Diagnostics.Process.csproj">
-      <SupportedFramework>net461;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net461;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
Similar to FileVersionInfo: APIs aren't WACK compliant so we were previously blocking netcore50 with 
<NotSupportedOnTargetFramework Include="netcore50"/>